pkg/cmd/hgctl/configBootstrap.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hgctl
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+func bootstrapConfigCmd() *cobra.Command {
+	configCmd := &cobra.Command{
+		Use:     "bootstrap <pod-name>",
```

Comment:
Higress is focused on gateways, maybe we don't need to specify pod name? Just select the random pod of higress gateway?

Reply:
This is a operating tools, which means operators/developers may debug some certain higress gateway pod, like if the number of replicas is more than one. A random pod is not a good idea to debug.

Reply:
A better idea is to make this optional. If the pod name is empty, config retrieves from one of the higress gateway pods in a namespace.